akka-remote/src/main/scala/akka/remote/artery/compress/InboundCompressions.scala

```diff
@@ -293,7 +293,7 @@ private[remote] abstract class InboundCompression[T >: Null](
 
   private[this] val cms = new CountMinSketch(16, 1024, System.currentTimeMillis().toInt)
 
-  log.debug("Initializing {} for originUid [{}]", Logging.simpleName(getClass), originUid)
+  log.debug("[inbound-compression] Initializing {} for originUid [{}]", Logging.simpleName(getClass), originUid)
```

Comment:
better to create the logger correctly, it's currently wrong class
```
val _inboundActorRefsLog = Logging(system, classOf[InboundManifestCompression])
```

Reply:
Good point :)

Reply:
it's fine actually, no need to change; was just more helpful for me to look for those markers when debugging